src/sentry/ingest/userreport.py

```diff
@@ -33,6 +33,16 @@ def save_userreport(
 ):
     with metrics.timer("sentry.ingest.userreport.save_userreport"):
         if is_org_in_denylist(project.organization):
+            metrics.incr("user_report.create_user_report.filtered", tags={"reason": "org.denylist"})
+            logger.info(
+                "Filtered an org in the deny list",
+                extra={
+                    "organization_id": project.organization.id,
+                    "organization_slug": project.organization.slug,
+                    "project_id": project.id,
+                    "project_slug": project.slug,
+                },
```

Comment:
Could we get away with the metric only for now? I wonder if this would cost a ton as we can get millions of dropped data from spammy automated reports (e.g: unreal engine)